Source/DafnyDriver/BoogieXmlConvertor.cs

```diff
@@ -111,7 +115,7 @@ public static class BoogieXmlConvertor {
         EndTime = DateTimeOffset.Parse(endTime)
       };
 
-      if (outcome == "correct") {
+      if (outcome is "correct" or "valid") {
```

Comment:
This isn't necessary any more, as an earlier change made `TestResultForSplit` a separate method.